x-pack/dockerlogbeat/docs/troubleshooting.asciidoc

```diff
@@ -0,0 +1,35 @@
+[[log-driver-troubleshooting]]
+[role="xpack"]
+== Troubleshooting
+
+experimental[]
+
+// TODO: Add details about locating the log files written by the Elastic Log
```

Comment:
On Linux, it will end up in the same place as the rest of docker logs, usually the system journal. On MacOS, it's a different story. I had to open a `screen` session to the hyperkit VM used by docker on mac.

Reply:
From the readme:

```
First, you need to shell into the VM that `runc` lives in. To do this, you need to find the tty for the VM. On newer versions of Docker For Mac, it's at: `~/Library/Containers/com.docker.docker/Data/vms/0/tty` On older versions, it's _somewhere_ else in `~/Library/Containers/com.docker.docker`. 


Once you find it, just run `screen ~/Library/Containers/com.docker.docker/Data/vms/0/tty`


The location of the logs AND the container base directory in the docker docs is wrong.


You can use this to find the list of plugins running on the host: `runc --root /containers/services/docker/rootfs/run/docker/plugins/runtime-root/plugins.moby/ list`

The logs are in `/var/log/docker`. If you want to make the logs useful, you need to find the ID of the plugin. Back on the darwin host, run `docker plugin inspect $name_of_plugin | grep Id` use the hash ID to grep through the logs: `grep 22bb02c1506677cd48cc1cfccc0847c1b602f48f735e51e4933001804f86e2e docker.*`
```

Reply:
What about Windows?

Reply:
I'm just going to point to the readme file and we can update that with more details. I think the info here is too complicated and weird to publish in the docs. :-)